.github/workflows/cron.yaml

```diff
@@ -0,0 +1,60 @@
+name: Update VEX Hub
+on:
+  schedule:
+    - cron: "0 0 * * *" # Every day
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  actions: read
+
+jobs:
+  update:
+    name: Update VEX Hub
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't see any use for this env.
Can you explain why we need this?

Reply:
oops 😄 
Perhaps i wanted to use `env.GITHUB_TOKEN`, but still used `secrets.ORG_REPO_TOKEN` in `actions/checkout`.

So we can remove this 😄

Reply:
Removed https://github.com/aquasecurity/vexhub/pull/6/commits/1f889dec7cf9a390d8e93c8996df343652c79eba